modules/manage/partials/tiered-storage.adoc

```diff
@@ -1133,6 +1133,35 @@ rpk topic alter-config <topic_name> --set redpanda.remote.read=true
 
 See also: xref:{topic-recovery-link}[Topic Recovery], xref:manage:kubernetes/k-remote-read-replicas.adoc[Remote Read Replicas]
 
+== Safe pause and resume
+
+When running Tiered Storage, you can safely pause and resume uploads to cloud storage without risking data consistency or loss. To pause or resume segment uploads to cloud storage, use the `cloud_storage_enable_segment_uploads` configuration property (default is `true`), which allows segment uploads to proceed normally after the pause completes and uploads resume.
+
+While uploads are paused, data accumulates locally, which can lead to full disks if the pause is prolonged. In such cases, Redpanda throttles produce requests, and rejects new Kafka produce requests to prevent data from being written. Additionally, this pauses cloud storage housekeeping, meaning segments are neither uploaded nor removed from cloud storage. However, it is still possible to consume data from cloud storage when you have paused uploads.
+
+When you set `cloud_storage_enable_segment_uploads` to `false`, all in-flight segment uploads will complete, but no new segment uploads will commence until after the value is set back to `true`. During this pause, Tiered Storage enforces consistency by ensuring that no segment in local storage is deleted until it is successfully uploaded to cloud storage. This means that when uploads are resumed, no user intervention is needed, and no data gaps are created.
+
+Use the `redpanda_cloud_storage_paused_archivers` metric to monitor the status of paused uploads. It displays a non-zero value whenever uploads are paused.
+
+NOTE: Do not use `redpanda.remote.read` or `redpanda.remote.write` to pause and resume segment uploads. Doing so can lead to a gap between local data and the data in the cloud storage that cannot be re-enabled without additional user intervention. Also, in such cases, it is possible that the oldest segment is not aligned with the last uploaded segment due to the gap.
```

Comment:
> that cannot be re-enabled without additional user intervention.

I want to avoid giving end users the impression that segment uploads can be resumed via `redpanda.remote.write` with "some" level of user intervention, as this would lead to increased support costs. Can we explicitly mention it is not supported?